src/components/FeatureViews/Courses/ClassEnrollmentRow.js

```diff
@@ -0,0 +1,363 @@
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import { ResponsiveButton } from "../../../theme/ThemedComponents/Button/ResponsiveButton";
+
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import MailOutlineIcon from "@material-ui/icons/MailOutline";
+import ChatOutlinedIcon from "@material-ui/icons/ChatOutlined";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { USER_TYPES, fullName } from "../../../utils";
+import { omouBlue, highlightColor } from "../../../theme/muiTheme";
+import IconButton from "@material-ui/core/IconButton";
+import MobileMenu from "@material-ui/icons/MoreVert";
+
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/es/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Divider from "@material-ui/core/Divider";
+
+import Accordion from "@material-ui/core/Accordion";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import AccessControlComponent from "../../OmouComponents/AccessControlComponent";
+import StudentEnrollmentBackground from "./ClassEnrollmentBackground";
+
+const useStyles = makeStyles((theme) => ({
+  table: {
+    minWidth: 1460,
+    [theme.breakpoints.between("md", "lg")]: {
+      minWidth: 910,
+    },
+    [theme.breakpoints.between("sm", "md")]: {
+      minWidth: 677,
+    },
+  },
+  menuSelected: {
+    "&:hover": { backgroundColor: highlightColor, color: "#28ABD5" },
+    "&:focus": { backgroundColor: highlightColor },
+  },
+  center: {
+    position: "relative",
+    top: "-15px",
+  },
+  dropdown: {
+    border: "1px solid #43B5D9",
+    borderRadius: "5px",
+  },
+  noBorderBottom: {
+    borderBottom: "none",
+  },
+  accordionNotes: {
+    textAlign: "left",
+    display: "inline-block",
+    marginTop: "10px",
+  },
+  accordionNotesBorder: {
+    border: "1px #E0E0E0 solid",
+    borderRadius: "25px",
+    margin: "0px 24px 25px 24px",
+    paddingTop: "0px",
+  },
+  studentRenderAccordionSpacing: {
+    width: "202px",
+  },
+  parentRenderAccordionSpacing: {
+    width: "200px",
+  },
+  actionsRenderAccordionSpacing: {
+    width: "200px",
+  },
+  iconRenderAccordionSpacing: {
+    position: "relative",
+    left: "6.5em",
+  },
+  studentInfoSpacing: {
+    margin: "15px",
+  },
+}));
+
+const ClassEnrollmentRow = ({
+  fullStudentName,
+  accountType,
+  studentId,
+  studentInfo,
+  parentAccountType,
+  parentId,
+  concatFullParentName,
+  phoneNumber,
+  handleOpenModal,
+  enrollmentList,
+  enrollmentID,
+  courseTitle
+}) => {
+  const { location } = useHistory();
+  const paramsID = useParams();
+  const dispatch = useDispatch();
+
+  let courseID;
+
+  paramsID.id ? (courseID = paramsID.id) : (courseID = paramsID.courseID);
```

Comment:
combine lines 106 and 108:
```
const courseId = paramsID.id ? paramsID.id : paramsID.courseID
```

Reply:
Done! I need this drilled into my brain - this is what I want to be better at!